Sources/main.swift

```diff
@@ -18,18 +18,18 @@ func showUsage() {
 	)
 }
 
-let arguments = Array(CommandLine.arguments.dropFirst(1))
+let arguments = CommandLine.arguments.dropFirst(1)
 
-switch arguments.stringAt(0) {
-case "--help":
+switch arguments.first {
+case "--help"?:
```

Comment:
I initially had it like this, but I foolishly used `switch arguments[0]` which didn't work. Thanks for showing me the correct way.